numba_dpex/core/datamodel/models.py

```diff
@@ -54,6 +59,21 @@ def __init__(self, dmm, fe_type):
         super(ArrayModel, self).__init__(dmm, fe_type, members)
 
 
+class ExecutionQueueModel(ListModel):
```

Comment:
Why `ListModel`?

Reply:
We are not decided yet, just a place holder for now.